wbooks/App.js

```diff
@@ -15,17 +15,14 @@ import {
   StatusBar,
 } from 'react-native';
 
-import {
-  Header,
-  Colors,
-} from 'react-native/Libraries/NewAppScreen';
 import BookList from './src/app/screens/booklist';
+
```

Comment:
Delete this i will explain it to you why on Slack

Reply:
done!